guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
+
+*A survey of 404 users and non-users of Bitcoin about the roles of trust, self-efficacy, and risk on adoption intention.*
```

Comment:
The current summary doesn't say much about what the paper found but merely states what it did.  Generally the final sentence in the abstract of research papers will give a good summary of the research aims. Below is copied from the paper and clearly explains what the results found not just outline what they aimed to do as the previous summary did. 

```suggestion
Results revealed that trust is a critical factor affecting adoption intention, with perceived self-efficacy having a mediating effect. Building on the results, practical recommendations are offered that could lower the entry barriers and facilitate the adoption of crypto-assets.
```

Reply:
I did not see any practical recommendations in the paper. Did I miss them? Also not sure what a "mediating effect" is in this case. Overall, I found the language in the paper (and many of the other papers) overly complicated and tedious to read, so I avoided copy/pasting.